ci/builder/requirements.txt

```diff
@@ -35,6 +34,7 @@ psutil==5.9.4
 pydantic==1.10.4
 pyelftools==0.29
 PyMySQL==1.0.2
+pyright==1.1.322
```

Comment:
I'm not wild about this wrapper script, which installs its own version of Node and NPM and all sorts of other crazy things. We need to be in charge of our own destiny. We should:

  * Install Node, npm, and pyright in the ci-builder Docker image
  * Use `npx pyright@1.1.322` in the `pycheck` script to run pyright
  * Update the developer guide to indicate the minimum supported version of Node, and provide installation instructions for macOS and Debian
  * Announce in #eng-announce this new dependency if editing Python code in this repository
  * (Stretch) Update the Nix flake in misc to install the right version of Node/npm/pycheck

That means CI will use a pinned and preinstalled version of pyright; individual developers on their own machines will be prompted by npx to install the right version of pyright (assuming they've already installed Node/npm on their machines).

Reply:
Sounds good -- I'll update